cmd/maesh/maesh.go

```diff
@@ -104,6 +105,8 @@ func maeshCommand(iConfig *cmd.MaeshConfiguration) error {
 		APIHost:          iConfig.APIHost,
 		MinTCPPort:       minTCPPort,
 		MaxTCPPort:       minTCPPort + iConfig.LimitTCPPort,
+		MinUDPPort:       minUDPPort,
```

Comment:
I am slightly bothered that we don't always list TCP, UDP and HTTP in the same order :p

To be consistent with the dashboard (maybe?) WDYT of always going HTTP>TCP>UDP?

Reply:
Sure :)